psalm.xml

```diff
@@ -68,5 +68,12 @@
                 <file name="tests/AbstractLexerTest.php" />
             </errorLevel>
         </RedundantConditionGivenDocblockType>
+        <MissingClassConstType>
+            <errorLevel type="suppress">
+                <!-- cannot fix that because we support PHP 8.1+ -->
+                <file name="tests/ConcreteLexer.php" />
+                <file name="tests/AbstractLexerTest.php" />
+            </errorLevel>
+        </MissingClassConstType>
```

Comment:
Let's ignore this error everywhere? I don't want to maintain that list.

Reply:
Fixed